nifi-nar-bundles/nifi-hive-bundle/nifi-hive-processors/src/main/java/org/apache/nifi/dbcp/hive/HiveConnectionPool.java

```diff
@@ -150,6 +154,17 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .build();
 
+    public static final PropertyDescriptor VALIDATION_QUERY_TIMEOUT = new PropertyDescriptor.Builder()
+            .name("Validation-query-timeout")
+            .displayName("Validation Query Timeout")
+            .description("The maximum amount of time allowed for running a validation query, "
+                    + "zero means there is no limit. Max time less than 1 second will be equal to zero.")
```

Comment:
In general the DataSource method is expecting an integer number of seconds right? So instead of "max time < 1 = 0" I think it's more accurate to say the time will be converted (and truncated) to an integer number of seconds.